django_tables2/rows.py

```diff
@@ -71,12 +71,20 @@ def table(self):
         '''
         return self._table
 
+    def get_even_odd_css_class(self):
+        '''
+        Return css class `even` or `odd`
+        '''
+        if next(self._table._counter) % 2 == 0:
```

Comment:
Binary notation is more efficient but is less readable for other people.
PEP 20 -- The Zen of Python:
- Simple is better than complex.
- Readability counts.